grammar.y

```diff
@@ -341,7 +341,7 @@ table_alias:
   }
 | STRING
   {
-    $$ = Identifier(string($1))
+    $$ = Identifier(string($1[1:len($1)-1]))
```

Comment:
This is the bug discovered with the test . 

Bug explanation:
According to the `table_alias` grammar rule, a table alias can be an `identifier` (`SELECT * FROM table t`) or `string` (`SELECT * FROM table 't'`). If it is a string, we have to remove the single quotes because that's not part of the name of the alias. Using `SELECT * FROM table 't'` as an example, the value of `$1` would `'t'` in the above grammar. But we only pass `t` to the `Identifier` node of the AST.

Reply:
Maybe:
`%token <bytes> IDENTIFIER STRING INTEGRAL HEXNUM FLOAT BLOBVAL`
The `STRING` token has this assumption since the lexer has that rule to generate the token?